src/analyses/raceAnalysis.ml

```diff
@@ -3,6 +3,85 @@
 open GoblintCil
 open Analyses
 
+(** Data race analysis with tries and hookers.
```

Comment:
I would guess it's a Futurama-inspired pun: https://en.wiktionary.org/wiki/with_blackjack_and_hookers

(but I agree let's we have a chuckle about it internally, and then remove it to avoid someone who doesn't know that getting offended)